server/controllers/payroll/multiplePayroll/calculation.js

```diff
@@ -25,4 +25,9 @@ function iprTax(annualCumulation, iprScales) {
   return iprValue;
 }
 
+function automaticRubric(coefficient, variables) {
```

Comment:
Just FYI - if you use `...variables`, it will automatically create an array for you.